File: PLAYERTITILES/playertitles.py
import requests

r = requests.get('https://valorant-api.com/v1/playertitles?language=ko-KR')
kor = r.json()
b = kor['data']
all = len(b)
counts = 0
for counts in range(0, all):
    korname = b[counts]['displayName'].upper()
    print(f'{korname}\n{counts + 1}/{all}')

File: PLAYERTITILES/playertitles.js
const axios = require('axios');

axios.get('https://valorant-api.com/v1/playertitles?language=ko-KR', {'User-Agent': 'info'}).then((Response)=>{
    var data = Response.data['data']
    for (var i = 0; i < data.length; i++) {
        var korname = data[i]['displayName']
        if (korname != ' '){
            console.log(korname)
        }
        else {
            console.log('칭호 없음')
        }
        console.log(i+1 + '/' + data.length)
    }
}).catch((Error)=>{
    console.log(Error);
}) 
